packs/feats/demolishing-knockback.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You make a forceful attack that has the power to push your foe through walls. Spend a Mythic Point and make a Strike at mythic proficiency. If you hit and deal damage to the target, you push them up to 15 feet away from you. If this movement would cause the target to collide with an object that has a hardness of 5 or less, the object breaks and the target is pushed through the object's space for the remainder of the forced movement, taking an additional @Damage[2d6[bludgeoning]] damage (the GM can decide that this is a different type of damage depending on the object the target was pushed through; for instance, slashing damage is the target is pushed through a pane of glass). If the forced movement doesn't cause the target to collide with an object, the target must succeed at a Reflex saving throw against your class DC or be knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone] at the end of the movement.</p>"
+            "value": "<p>You make a forceful attack that has the power to push your foe through walls. Spend a Mythic Point and make a Strike at mythic proficiency. If you hit and deal damage to the target, you push them up to 15 feet away from you. If this movement would cause the target to collide with an object that has a hardness of 5 or less, the object breaks and the target is pushed through the object's space for the remainder of the forced movement, taking an additional @Damage[2d6[bludgeoning]] damage (the GM can decide that this is a different type of damage depending on the object the target was pushed through; for instance, @Damage[2d6[slashing]]{slashing} damage is the target is pushed through a pane of glass).</p>\n<p>If the forced movement doesn't cause the target to collide with an object, the target must succeed at a @Check[reflex|against:class] saving throw against your class DC or be knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone] at the end of the movement.</p>"
```

Comment:
```suggestion
            "value": "<p>You make a forceful attack that has the power to push your foe through walls. Spend a Mythic Point and make a Strike at mythic proficiency. If you hit and deal damage to the target, you push them up to 15 feet away from you. If this movement would cause the target to collide with an object that has a hardness of 5 or less, the object breaks and the target is pushed through the object's space for the remainder of the forced movement, taking an additional @Damage[2d6[bludgeoning]] damage (the GM can decide that this is a different type of damage depending on the object the target was pushed through; for instance, @Damage[2d6[slashing]]{slashing} damage if the target is pushed through a pane of glass).</p>\n<p>If the forced movement doesn't cause the target to collide with an object, the target must succeed at a @Check[reflex|against:class] saving throw against your class DC or be knocked @UUID[Compendium.pf2e.conditionitems.Item.Prone] at the end of the movement.</p>"
```